src/extensions.ts

```diff
@@ -0,0 +1,2 @@
+export const js: string[] = ['js', 'cjs', 'mjs', 'jsx'];
```

Comment:
```suggestion
export const js = ['js', 'cjs', 'mjs', 'jsx'] satisfies string[];
```

Not sure if people exactly care about what's in here but satisfies gives us slightly better types so they don't need to dive into the source?